huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_cassandra_instance.go

```diff
@@ -487,7 +495,8 @@ func resourceGeminiDBInstanceV3Read(d *schema.ResourceData, meta interface{}) er
 	d.Set("security_group_id", instance.SecurityGroupId)
 	d.Set("dedicated_resource_id", instance.DedicatedResourceId)
 	d.Set("mode", instance.Mode)
-	d.Set("db_user_name", instance.DbUserName)
```

Comment:
Cannot obtain DB user name from the API response?

Reply:
that was a mistake.